.github/workflows/tests.yml

```
@@ -29,13 +29,14 @@ jobs:
 
       - name: Install dependencies
         run: |
-          python -m pip install --upgrade pip
+          python -m pip install --upgrade pip setuptools

```

```suggestion
          python -m pip install --upgrade pip 

```

I dont think you need this here. Probably only in Plugin action?